From: <ohering@suse.de>
Date: Fri, 27 Feb 2015 11:25:59 -0800
Patch-mainline: never, kabi
Subject: kabi: clockevents: export clockevents_unbind again

Kernel v4.1-rc1, commit 32a158325acf12842764b1681f53903673f2f22e exported
the correct public function. Unfortunately the static variant made it into
the kabi list. Just reexport it.

--- a/kernel/time/clockevents.c
+++ b/kernel/time/clockevents.c
@@ -371,6 +371,7 @@
 	mutex_unlock(&clockevents_mutex);
 	return ret;
 }
+EXPORT_SYMBOL_GPL(clockevents_unbind);
 EXPORT_SYMBOL_GPL(clockevents_unbind_device);
 
 /**
